backend/internal/service/server.go

```diff
@@ -82,6 +83,11 @@ func setupRoutes(app *fiber.App, config config.Config) {
 		r.Patch("/:recommendationId", recommendationHandler.ReactToRecommendation)
 	})
 
+	playlistHandler := playlist.NewHandler(repository.Playlist)
+	app.Route("/playlist", func(r fiber.Router) {
+		r.Post("/on_queue/:id", playlistHandler.AddToUserOnQueue)
```

Comment:
this is subtle but can we call the parameter `:userId` so it's clear that it's not the trackId? (make sure to update it in the handler too)

Reply:
Changed this!